CHANGELOG.md

```diff
@@ -13,6 +13,51 @@ Branch: `nilrt/master/hardknott`
 
 ----
 
+## 9.1
+Branch: `nilrt/22.8/hardknott`
+
+### nilrt
+
+#### Added
+- Added NILRT GRUB version 22.8 to dist feed
+- Added scripts to diff feeds between NILRT releases
+- Added configuration to use an NI-internal hashserver for builds
+- Added script to automate merges from upstream
+- Added workflow to sync GitHub issues to AzDO
```

Comment:
Let's keep this repo maintenance stuff out of the changelog, just because it probably isn't useful to end-users.